airqo-platform/AirQo-api
# Iot core bridges

- What does this PR do?
This PR created new endpoints for publishing telemetry events to the IoT core.
- What JIRA task is related to this PR?
There is no JIRA link right now, we are still sorting out the use cases and identifying what the final product is.
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/device-registry` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run test` to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
partial url: /api/v1/devices/mqtt/config/gcp [GET]
partial url: /api/v1/devices/mqtt/config/gcp [PUT]
partial url: /api/v1/devices/http/config/gcp [GET]
partial url: /api/v1/devices/http/config/gcp [PUT]
More details about how to utilize these end point can be found under the **Device Registry** sheet/tab of the [API documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=897606332)
